routes/error-tracker.js

```diff
@@ -28,6 +26,10 @@ const logName = 'javascript.errors';
 const SERVER_START_TIME = Date.now();
 const errorsToIgnore = ['stop_youtube',
   'null%20is%20not%20an%20object%20(evaluating%20%27elt.parentNode%27)'];
+const location = '([^ \\n]+):(\\d+):(\\d+)';
```

Comment:
Can we have a better name for this? "Location" usually refers to [Window.location](https://developer.mozilla.org/en-US/docs/Web/API/Window/location). Perhaps `chromeStackFunction`?